pytest_mypy_plugins/utils.py

```diff
@@ -260,7 +262,9 @@ def format_error_lines(lines: List[str]) -> str:
             format_error_lines(actual_message_lines), format_error_lines(expected_message_lines)
         )
 
-        if (
+        if expected_line and expected_line.regex:
+            error_message += "The actual output does not match the expected regex."
```

Comment:
Do we cover this line? 🤔

Reply:
It's covered everytime we have `regex: yes` in tests but there is no way to test the actual output of `pytest-mypy-plugins` as far as I know.